server/routes/ad.ts

```diff
@@ -220,38 +233,88 @@ export default class AdService {
         .callAsCurrentUser('ad.getDetector', {
           detectorId,
         });
+
+      // Considering any detector with no detection date range to be a realtime detector
```

Comment:
Get detector API will also return detector type https://github.com/opendistro-for-elasticsearch/anomaly-detection/blob/master/src/main/java/com/amazon/opendistroforelasticsearch/ad/model/AnomalyDetectorType.java#L21

I'm ok to use either detection date range or this type field.

Reply:
I see, thanks. I think I will leave as-is for now, to prevent adding more frontend data types and to keep consistent with the alerting changes.

Reply:
Sure, sounds good